fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/steps/FintechStagesUtils.java

```diff
@@ -0,0 +1,28 @@
+package de.adorsys.fintech.tests.e2e.steps;
+
+import io.restassured.RestAssured;
+import io.restassured.specification.RequestSpecification;
+
+import java.util.UUID;
+
+public class FintechStagesUtils {
+
+    public static final String X_REQUEST_ID = "X_REQUEST_ID";
+    public static final String PIN_VALUE = "1234";
+    public static final String USERNAME = "tom";
```

Comment:
You need to generate a random user name on each test run - and it is easy to do so with FinTech server - just generate random user name